src/site/content/en/blog/module-workers/index.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Threading the Web with Module Workers"
```

Comment:
Web.dev uses sentence case for titles (unlike what your writing teachers taught you). This should be 'Threading the web with module workers'.

Please check your section titles throughout.

Reply:
Sounds good - only concern is that "Workers" is typically capitalized, because it's a reference to the feature named Web Workers. I was taking "Module Workers" to be a feature name here, though that is admittedly something I've only seen done colloquially - there is no specification called Module Workers, only a JS Modules addendum to the Web Workers specification.